src/engine/routing_algorithms/routing_base_ch.cpp

```diff
@@ -139,7 +139,124 @@ void search(const datafacade::ContiguousInternalMemoryDataFacade<algorithm::CH>
     }
 }
 
-// assumes that heaps are already setup correctly.
+double getPathDistance(const datafacade::ContiguousInternalMemoryDataFacade<Algorithm> &facade,
```

Comment:
yes, it is move one scope level up in the map matching branch. i would keep it as only CH-related in this PR